specifyweb/frontend/js_src/lib/components/QueryBuilder/Results.tsx

```diff
@@ -393,7 +400,8 @@ export function useFetchQueryResults({
   const resultsRef = React.useRef(results);
   const handleSetResults = React.useCallback(
     (results: RA<QueryResultRow | undefined> | undefined) => {
-      const filteredResults = results !== undefined ? filterArray(results) : undefined
+      const filteredResults =
+        results !== undefined ? filterArray(results) : undefined;
```

Comment:
no negated conditions - swap branches instead
or even better, just use `f.maybe(results, filterArray);`

Reply:
the one time it was obvious to use f.maybe, and I didn't :)